dom.bs

```diff
@@ -731,7 +755,7 @@ To <dfn export for=Event id=concept-event-initialize>initialize</dfn> an
  <a>canceled flag</a>.
  <li>Set the {{Event/isTrusted}} attribute
  to false.
- <li>Set the {{Event/target}} attribute to
+ <li>Set the {{Event/target}} and {{Event/srcElement}} attributes to
```

Comment:
An alternative might be that we actually define Event's "target" as a concept and set that here and than have the target and srcElement attributes reflect that.

Reply:
Filed a follow up issue for that: https://github.com/whatwg/dom/issues/570